src/main/java/uk/gov/ons/ctp/response/action/service/ActionPlanJobService.java

```diff
@@ -111,6 +100,7 @@ private boolean hasActionPlanBeenRunSinceLastSchedule(ActionPlan actionPlan) {
    * @param actionPlan Action plan to create and execute job for
    * @return ActionPlanJob that was created or null if it has not been created.
    */
+  @Transactional
```

Comment:
This annotation won't actually do anything because the method - technically - isn't 'public' per se... it could be made private. What this means is that Spring won't be able to intercept the method call. You'll have to put `@Transactional` on one of the 'entry points' into this bean